fvm/src/syscalls/mod.rs

```diff
@@ -169,6 +169,7 @@ pub fn bind_syscalls(
 
     linker.bind("debug", "log", debug::log)?;
     linker.bind("debug", "enabled", debug::enabled)?;
+    linker.bind("debug", "capture_artifact", debug::capture_artifact)?;
```

Comment:
nit: `save_artifact` or `write_artifact` or `store_artifact` etc. The artifact has already been created, so we're not really "capturing" anything.